File: Java Back-end/EasySaleApi1/Telas/Dominio/Cliente.java
package Dominio;



public class Cliente {
	
	public Integer getId() {
		return id;
	}
	public void setId(Integer id) {
		this.id = id;
	}
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	
	public String getdT_Nascimento() {
		return dT_Nascimento;
	}
	public void setdT_Nascimento(String DT_Nascimento) {
		this.dT_Nascimento = DT_Nascimento;
	}
	
	public String getBairro() {
		return Bairro;
	}
	public void setBairro(String bairro) {
		Bairro = bairro;
	}
	public String getEndereco() {
		return Endereco;
	}
	public void setEndereco(String endereco) {
		Endereco = endereco;
	}

	private Integer id;
	private String nome;
	private String email;
	public String dT_Nascimento;
	private String Bairro;
	private String Endereco;
	
}

File: C# API/WebAPIA/Models/Cliente.cs
using Dapper.Contrib.Extensions;
using System;

namespace WebAPI.Models
{
    [Table("Cliente")]
    public class Cliente : BaseModel
    {
        [Key]
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Endereco { get; set; }
        public string Bairro { get; set; }
        public string Email { get; set; }
        public string DT_Nascimento { get; set; }
    }
}
